src/components/NavBar/NavBar.tsx

```diff
@@ -13,100 +13,152 @@ import { useAuth } from "../../providers/AuthContext.tsx";
 import redirectWithPost from "../../util/redirectWithPost.ts";
 
 const NavBar: React.FC = () => {
-	const navigate = useNavigate();
-	const { user } = useAuth();
+  const navigate = useNavigate();
```

Comment:
repoyu VSCode'da açtığında uyarı gösteriyor olması lazım da, Prettier extensionunu yükler misin acaba formatlamanın her yerde aynı olması için?